app/lib/common/widgets/search_widget.dart

```diff
@@ -18,7 +18,7 @@ class SearchWidget extends StatelessWidget {
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 15),
       child: SearchBarAnimation(
```

Comment:
We don't need this animation, it doesn't look so good. And also, micro-interactions aren't priority, so this package can be discarded. We can just use the generic ```SearchWidget``` I've set up (came from news PR).

Reply:
Alright , the widget I created was `search.dart` . Let me check out the changes you've made .